### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by Helen-arch**
Screenshot 2023-12-14 at 21 07 48
Screenshot 2023-12-14 at 21 08 09

### CODE REVIEW

**Review by sviatikkondur — APPROVED** [review]
Great job! Just use search params to store pagination data in next PR

**Review by BetterSol — APPROVED** [review]
Looks great!

File: src/modules/CatalogPage/CatalogPage.tsx [hunk 1]
```diff
@@ -1,22 +1,71 @@
...
 import { Phone } from '../../types/Phone';
+import { Option } from '../../types/Option';
 import { ProductsList } from '../shared/components/ProductsList';
+import { CustomSelect } from '../shared/components/CustomSelect/CustomSelect';
+import { Pagination } from '../shared/components/Pagination/Pagination';
+
+const sortOptions: Option[] = [
+  { value: 'age', label: 'Newest' },
+  { value: 'title', label: 'Alphabetically' },
+  { value: 'price', label: 'Cheapest' },
```
Comment (Clover247) [inline src/modules/CatalogPage/CatalogPage.tsx:17]: Can we add a reverse sort like oldest, most expensive?

File: src/modules/shared/components/CustomSelect/CustomSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import { Option } from '../../../../types/Option';
+
+type Props = {
+  label: string;
+  options: Option[];
+  defaultOptionId: number;
+  onSelectAmount?: (selectedOption: OnChangeValue<Option, false>) => void;
+};
+
+const customStyles = {
```
Comment (Clover247) [inline src/modules/shared/components/CustomSelect/CustomSelect.tsx:12]: 🔥

**Review by Clover247 — APPROVED** [review]
Just a masterpiece

**Pull request merged**